app/views/thankYou.scala.html

```diff
@@ -2,21 +2,23 @@
 @import helper.CSRF
 
 @(
-    title: String,
-    id: String,
-    js: String,
-    description: Option[String] = None
+        title: String,
+        id: String,
+        js: String,
+        email: Option[String],
+        showSubCopy: Boolean = false
```

Comment:
What would the `showSubCopy` refer to? If it is related to subscriptions I would put it explicitly.

Reply:
It is actually sub in the newspaper sense but I agree it wasn't particularly clear. I've changed it to `showConfirmationEmailCopy` which should be much clearer